dozer/__main__.py

```diff
@@ -17,7 +17,7 @@
         'key': 'Put TOA API key here',
         'app_name': 'Dozer',
     },
-    'db_url': 'sqlite:///dozer.db',
+    'db_url': 'postgres://POSTGRES URL HERE',
```

Comment:
While this does technically fix the problem, the problem is that if there's any error in our `db_init` script, the config file will still fail to generate at all since it runs before the file is saved. See line 33 in `__main__.py`.